Bottom: 16430c6083356e18a759f841d8e53365e26fc5b1
Top:    2a5d8ed8ebb8ca9fa4595c6a65c29e0f32ec3ef1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-08 01:30:26 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 772fa9e..420416c 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -323,13 +323,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
